system/draft/test_upgrade.py

```diff
@@ -112,18 +126,18 @@
     # print(res)
     # assert res['op'] == 'REPLY'
 
-    await asyncio.sleep(7*5*60)
+    await asyncio.sleep(25*5*60)
     # await asyncio.sleep(180)
 
-    docker_7_hosts = [
-        testinfra.get_host('docker://node' + str(i)) for i in range(1, 8)
-    ]
-    # aws_25_hosts = [
-    #     testinfra.get_host('ssh://persistent_node'+str(i), ssh_config='/home/indy/.ssh/config') for i in range(1, 26)
+    # docker_7_hosts = [
```

Comment:
Does it make sense to have 2 tests, one for 7 nodes Docker pool and one for 25 nodes AWS pool

Reply:
Yes, I will split it.